section/topic modelling (#7)
MrLongbottom/P9-Rapport: sections/method/topic modelling.tex

\subsection{Topic Modelling}\label{subsec:topicModelling}
The objective of topic modelling is to infer topics (collections of words) in a document set.

> The result consists of a topic-word distribution matrix $\varphi$, which for each topic gives a distribution of words belonging to said topic, and a document-topic distribution matrix $\theta$ which for each document gives a distribution of topics which the document belongs to.

Reviewer: ```suggestion
The result consists of a topic-word distribution matrix $\varphi$, which for each topic gives a distribution of words belonging to said topic, and a document-topic distribution matrix $\theta$ which for each document gives a distribution of topics to which the document belongs to.
```

> The result consists of a topic-word distribution matrix $\varphi$, which for each topic gives a distribution of words belonging to said topic, and a document-topic distribution matrix $\theta$ which for each document gives a distribution of topics to which the document belongs to.

Reviewer: ```suggestion
The result consists of a topic-word distribution matrix $\beta$, which for each topic gives a distribution of words belonging to said topic, and a document-topic distribution matrix $\theta$ which for each document gives a distribution of topics to which the document belongs to.
```

> The result consists of a topic-word distribution matrix $\varphi$, which for each topic gives a distribution of words belonging to said topic, and a document-topic distribution matrix $\theta$ which for each document gives a distribution of topics to which the document belongs to.

Reviewer: Det er blevet til beta :+1:

\subsubsection{\acrlong{LDA}}\label{subsec:lda}
\todo{cite lda} introduce \gls{LDA}, which has since become a staple within topic modelling.

> \gls{LDA} works under the assumption that documents are generated from a specific generative process, and then tries to reverse engineer this process.

Reviewer: ```suggestion
\gls{LDA} works under the assumption that documents are generated from a specific generative process, and it tries to reverse engineer this process.
```

Author: ```suggestion
\gls{LDA} works under the assumption that documents are generated from a specific generative process, and tries to reverse engineer this process.
```

> This process generates $K$ topics and $M$ documents containing $N_{i\dots M}$ words.

Reviewer: Jeg stemmer på den nederste

> This process generates $K$ topics and $M$ documents containing $N_{i\dots M}$ words.

Author: ```suggestion
This process generates $K$ topics and $D$ documents containing $N_{d}$ words, where $d$ is a document in $D$.
```

> This genertative process assumes that documents are random mixtures of latent topics, and that each topics is a distribution over all the words in the corpus.

Reviewer: ```suggestion
This generative process assumes that documents are random mixtures of latent topics and that each topic is a distribution over all the words in the corpus.
```

> The generative process has two Dirichlet distributions $Dir(\alpha)$ for the document-topic relation and $Dir(\beta)$ for the topic-word relation.

Author: ```suggestion
The generative process has two Dirichlet distributions $Dir(\alpha)$ for the document-topic relation and $Dir(\eta)$ for the topic-word relation.
```

\begin{figure}[h]
	\centering

> 	\includegraphics[width= 0.3 \textwidth]{figures/Smoothed_LDA.PNG}

Author: ```suggestion
	\includegraphics[width= 0.3 \textwidth]{figures/Smoothed_LDA.PNG}
	\todo{make blei version of figure.}
```

	\caption{Plate notation for \gls{LDA}. Boxes symbolize repeated processes, shaded elements are observed information. Image is from \url{https://en.wikipedia.org/wiki/Latent_Dirichlet_allocation}}
	\label{fig:lda}
\end{figure}

> 

Author: \todo{change figure to Blei version}

> From these Dirichlet distributions, we can sample two multinomial distributions: document-topic $\theta$ and topic-word $\varphi$.

Author: ```suggestion
From these Dirichlet distributions, we can sample two multinomial distributions: document-topic $\theta$ and topic-word $\beta$.
```

> From these dirichlets we can sample document-topic multinomial distributions $\theta$ and topic-word multinomial distributions $\varphi$.

Reviewer: ```suggestion
From these dirichlets, we can sample two multinomial distributions: document-topic $\theta$ and topic-word $\varphi$.
```

> These Dirichlet distributions are tuned with the hyperparameters $\alpha$ and $\beta$, which adjust the entropy of the sampled distributions.

Author: ```suggestion
These Dirichlet distributions are tuned with the hyperparameters $\alpha$ and $\eta$, which adjust the entropy of the sampled distributions.
```

> An $\alpha$ value near one causes each document to be distributed over almost all topics, while a $\alpha$ value near 0 causes each document to be distributed over only a few topics.

Reviewer 2: ```suggestion
An $\alpha$ value near 1 causes each document to be distributed over almost all topics, while an $\alpha$ value near 0 causes each document to be distributed over only a few topics.
```

> $\alpha$ near one causes each document to be distributed over almost all topics, while a $\alpha$ near 0 causes each document to be distributed over onlt a few topics.

Reviewer: ```suggestion
An $\alpha$ value near one causes each document to be distributed over almost all topics, while a $\alpha$ value near 0 causes each document to be distributed over only a few topics.
```

> Similarly $\beta$ will adjust how many words each topic contains.

Author: ```suggestion
Similarly $\eta$ will adjust how many words each topic contains.
```

> This also has the added consequence that a high $\alpha$ will make documents appear more similar, and a high $\beta$ will make topics appear more similar.

Author: ```suggestion
This also has the added consequence that a high $\alpha$ will make documents appear more similar, and a high $\eta$ will make topics appear more similar.
```

> Using $\theta$ and $\varphi$ we can sample concrete topics $Z$ from documents, and concrete words $W$ from topics.

Reviewer 2: ```suggestion
Using $\theta$ and $\varphi$, we can sample concrete topics $Z$ from documents, and concrete words $W$ from topics.
```

> Using $\theta$ and $\varphi$, we can sample concrete topics $Z$ from documents, and concrete words $W$ from topics.

Author: ```suggestion
Using $\theta$ and $\beta$, we can sample concrete topics $Z$ from documents, and concrete words $W$ from topics.
```
